cypress/support/pages/backoffice/product-measurement-unit/list/product-measurement-unit-list-repository.ts

```diff
@@ -0,0 +1,20 @@
+import { autoWired } from '@utils';
+import { injectable } from 'inversify';
+
+@injectable()
+@autoWired
+export class ProductMeasurementUnitListRepository {
+  getSearchSelector = (): string => '[type="search"]';
+
+  getCreateButtonSelector = (): string => '.btn-create';
+
+  getEditButtonSelector = (): string => '.btn-edit';
```

Comment:
Could you please try to find better selectors, like 

`'button[href*="/product-measurement-unit-gui/add"]'` , `'button[href*="/product-measurement-unit-gui/edit"]'`or similar. Classes might be not stable